Resources/Prototypes/_Midnight/Entities/Clothing/OuterClothing/coats.yml

```
@@ -37,4 +35,129 @@
-          Quantity: 30
\ No newline at end of file
+          Quantity: 30
+
+- type: entity
+  parent: ClothingOuterStorageBase
+  id: ClothingOuterCoatSmuggler
+  suffix: Chameleon, Holdout
+  name: Chatter Coat
+  description: Smuggler kit radio coat.
+  components:
+    - type: Sprite
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)